spec/index.html

```diff
@@ -1064,9 +1089,13 @@ <h3>RDF Dataset Comparison</h3>
         the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in |DG2|.</li>
       <li>The [=named graph=] (|n|, |G|) is in |NG1| if and only if
         there is a [=named graph=] (<var>n'</var>, <var>G'</var>) in |NG2| such that
-        |M|(|n|) = <var>n'</var> and
-        the triple (|s|, |p|, |o|) is in |G| if and only if
-        the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in <var>G'</var>.</li>
+        the following are true:
+        <ul>
+          <li>|M|(|n|) is [=RDF term equality|equal=] to <var>n'</var>.</li>
+          <li>The triple (|s|, |p|, |o|) is in |G| if and only if
```

Comment:
See https://github.com/w3c/rdf-concepts/pull/161#discussion_r1976135601
```suggestion
          <li>The triple (|s|, |p|, |o|) is in |G| and
```

Reply:
Leave as is.

 "if and only if" is a two way implication -- `<=>`. There are many "if and only if" in this document in definition sections.

Reply:
It is repeating the text from 'graph isomorphism', which I find helpful.

Reply:
> It is repeating the text from 'graph isomorphism', which I find helpful.

I have nothing against such repetition, but again, I find this phrasing problematic.

It does not communicate clearly to me.

Given that this text is repeated from 'graph isomorphism', I think it should be changed in both locations, as it is no less problematic at the other location.

Reply:
Changing the _"if and only if"_ to _"and"_ (as proposed by @TallTed ) is not correct because, then, it is not clear what the _s_,  _p_, and  _o_ are that this sub-bullet point is talking about.

Yet, I also see @TallTed's point that having the _"if and only if"_ within the _"such that"_ condition of another _"if and only if"_ is problematic; it may not be incorrect but at least confusing.

To fix this issue, I propose to qualify the inner _"if and only if"_ statement as follows:

> * The named graph (_n_, _G_) is in _NG1_ if and only if there is a named graph (_n'_, _G'_) in _NG2_ such that the following are true:
>   *  ...
>   * For every triple _t_=(_s_, _p_, _o_), it holds that _t_ is in _G_ if and only if the triple ( _M_(_s_), _M_(_p_), _M_(_o_) ) is in _G'_.

Hence, the edit suggestion here would then be:
 ```suggestion
           <li>For every triple |t|=(|s|, |p|, |o|), it holds that
             |t| is in |G| if and only if
```
What do you think?

Reply:
Thanks Ted. I have committed my proposed change. Are you okay with this PR now?